[
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     857     gcc: _configtest.c\n858     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     859         3 | #pragma clang diagnostic error \"-Wattributes\"\n     860           |\n     861     _configtest.c: In function '__declspec':\n  >> 862     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "863         5 | int __declspec(thread) foo;\n     864           |                        ^~~\n  >> 865     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     866         9 | {\n     867           | ^\n868     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     869         5 | int __declspec(thread) foo;\n     870           |     ^~~~~~~~~~\n  >> 871     _configtest.c:11: error: expected '{' at end of input\n     872        11 | }\n     873           |",
    "implicit-int]\n     869         5 | int __declspec(thread) foo;\n     870           |     ^~~~~~~~~~\n  >> 871     _configtest.c:11: error: expected '{' at end of input\n     872        11 | }\n     873           |\n874     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     875        11 | }\n     876           |\n     877     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     878         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "872        11 | }\n     873           |\n     874     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     875        11 | }\n     876           |\n877     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     878         3 | #pragma clang diagnostic error \"-Wattributes\"\n     879           |\n     880     _configtest.c: In function '__declspec':\n  >> 881     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "882         5 | int __declspec(thread) foo;\n     883           |                        ^~~\n  >> 884     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     885         9 | {\n     886           | ^\n887     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     888         5 | int __declspec(thread) foo;\n     889           |     ^~~~~~~~~~\n  >> 890     _configtest.c:11: error: expected '{' at end of input\n     891        11 | }\n     892           |",
    "implicit-int]\n     888         5 | int __declspec(thread) foo;\n     889           |     ^~~~~~~~~~\n  >> 890     _configtest.c:11: error: expected '{' at end of input\n     891        11 | }\n     892           |\n893     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     894        11 | }\n     895           |\n     896     failure.\n\n     ...",
    "...\n\n     2969          |                              ^~\n     2970    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2971    numpy/core/src/multiarray/scalartypes.c.src:2707:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2972     2707 |         if (ret->obmeta.base == -1) {\n     2973          |                              ^~\n     2974    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2994     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2995          |                              ^\n     2996    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2997    numpy/core/src/multiarray/scalartypes.c.src:594:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2998      594 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2999          |     ^~\n\n     ...",
    "...\n\n     3136          |                              ^~\n     3137    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3138    numpy/core/src/multiarray/scalartypes.c.src:2707:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3139     2707 |         if (ret->obmeta.base == -1) {\n     3140          |                              ^~\n     3141    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     857     gcc: _configtest.c\n858     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     859         3 | #pragma clang diagnostic error \"-Wattributes\"\n     860           |\n     861     _configtest.c: In function '__declspec':\n  >> 862     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "863         5 | int __declspec(thread) foo;\n     864           |                        ^~~\n  >> 865     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     866         9 | {\n     867           | ^\n868     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     869         5 | int __declspec(thread) foo;\n     870           |     ^~~~~~~~~~\n  >> 871     _configtest.c:11: error: expected '{' at end of input\n     872        11 | }\n     873           |",
    "implicit-int]\n     869         5 | int __declspec(thread) foo;\n     870           |     ^~~~~~~~~~\n  >> 871     _configtest.c:11: error: expected '{' at end of input\n     872        11 | }\n     873           |\n874     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     875        11 | }\n     876           |\n     877     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     878         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "872        11 | }\n     873           |\n     874     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     875        11 | }\n     876           |\n877     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     878         3 | #pragma clang diagnostic error \"-Wattributes\"\n     879           |\n     880     _configtest.c: In function '__declspec':\n  >> 881     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "882         5 | int __declspec(thread) foo;\n     883           |                        ^~~\n  >> 884     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     885         9 | {\n     886           | ^\n887     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     888         5 | int __declspec(thread) foo;\n     889           |     ^~~~~~~~~~\n  >> 890     _configtest.c:11: error: expected '{' at end of input\n     891        11 | }\n     892           |",
    "implicit-int]\n     888         5 | int __declspec(thread) foo;\n     889           |     ^~~~~~~~~~\n  >> 890     _configtest.c:11: error: expected '{' at end of input\n     891        11 | }\n     892           |\n893     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     894        11 | }\n     895           |\n     896     failure.\n\n     ...",
    "...\n\n     2969          |                              ^~\n     2970    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2971    numpy/core/src/multiarray/scalartypes.c.src:2707:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2972     2707 |         if (ret->obmeta.base == -1) {\n     2973          |                              ^~\n     2974    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2994     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2995          |                              ^\n     2996    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2997    numpy/core/src/multiarray/scalartypes.c.src:594:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2998      594 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2999          |     ^~\n\n     ...",
    "...\n\n     3136          |                              ^~\n     3137    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3138    numpy/core/src/multiarray/scalartypes.c.src:2707:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3139     2707 |         if (ret->obmeta.base == -1) {\n     3140          |                              ^~\n     3141    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     103     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 104     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 108     collect2: error: ld returned 1 exit status",
    ">> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 108     collect2: error: ld returned 1 exit status\n     109     failure.\n     110     removing: _configtest.c _configtest.o\n     111     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     112",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     161     gcc: _configtest.c\n>> 162     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     163         1 | #include <sys/endian.h>\n     164           |          ^~~~~~~~~~~~~~\n     165     compilation terminated.\n  >> 166     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "161     gcc: _configtest.c\n  >> 162     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     163         1 | #include <sys/endian.h>\n     164           |          ^~~~~~~~~~~~~~\n     165     compilation terminated.\n>> 166     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     167         1 | #include <sys/endian.h>\n     168           |          ^~~~~~~~~~~~~~\n     169     compilation terminated.\n     170     failure.\n     171     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     277     gcc: _configtest.c\n     278     _configtest.c: In function 'main':\n>> 279     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     280         7 |     (void) SIZEOF_LONGDOUBLE;\n     281           |            ^~~~~~~~~~~~~~~~~\n     282           |            SIZEOF_LONG_DOUBLE\n     283     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "280         7 |     (void) SIZEOF_LONGDOUBLE;\n     281           |            ^~~~~~~~~~~~~~~~~\n     282           |            SIZEOF_LONG_DOUBLE\n     283     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     284     _configtest.c: In function 'main':\n>> 285     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     286         7 |     (void) SIZEOF_LONGDOUBLE;\n     287           |            ^~~~~~~~~~~~~~~~~\n     288           |            SIZEOF_LONG_DOUBLE\n     289     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     423     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 424     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 425     collect2: error: ld returned 1 exit status\n     426     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 427     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 428     collect2: error: ld returned 1 exit status",
    ">> 425     collect2: error: ld returned 1 exit status\n     426     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 427     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 428     collect2: error: ld returned 1 exit status\n     429     failure.\n     430     removing: _configtest.c _configtest.o\n     431     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     432",
    "857     gcc: _configtest.c\n     858     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     859         3 | #pragma clang diagnostic error \"-Wattributes\"\n     860           |\n     861     _configtest.c: In function '__declspec':\n>> 862     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     863         5 | int __declspec(thread) foo;\n     864           |                        ^~~\n  >> 865     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     866         9 | {",
    "860           |\n     861     _configtest.c: In function '__declspec':\n  >> 862     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     863         5 | int __declspec(thread) foo;\n     864           |                        ^~~\n>> 865     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     866         9 | {\n     867           | ^\n     868     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     869         5 | int __declspec(thread) foo;",
    "866         9 | {\n     867           | ^\n     868     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     869         5 | int __declspec(thread) foo;\n     870           |     ^~~~~~~~~~\n>> 871     _configtest.c:11: error: expected '{' at end of input\n     872        11 | }\n     873           |\n     874     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     875        11 | }\n     876           |",
    "876           |\n     877     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     878         3 | #pragma clang diagnostic error \"-Wattributes\"\n     879           |\n     880     _configtest.c: In function '__declspec':\n>> 881     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     882         5 | int __declspec(thread) foo;\n     883           |                        ^~~\n  >> 884     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     885         9 | {",
    "879           |\n     880     _configtest.c: In function '__declspec':\n  >> 881     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     882         5 | int __declspec(thread) foo;\n     883           |                        ^~~\n>> 884     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     885         9 | {\n     886           | ^\n     887     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     888         5 | int __declspec(thread) foo;",
    "885         9 | {\n     886           | ^\n     887     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     888         5 | int __declspec(thread) foo;\n     889           |     ^~~~~~~~~~\n>> 890     _configtest.c:11: error: expected '{' at end of input\n     891        11 | }\n     892           |\n     893     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     894        11 | }\n     895           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     1223    gcc: _configtest.c\n     1224    _configtest.c: In function 'main':\n>> 1225    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1226        8 |     (void) HAVE_DECL_SIGNBIT;\n     1227          |            ^~~~~~~~~~~~~~~~~\n     1228          |            HAVE_DECL_ISNAN\n     1229    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1226        8 |     (void) HAVE_DECL_SIGNBIT;\n     1227          |            ^~~~~~~~~~~~~~~~~\n     1228          |            HAVE_DECL_ISNAN\n     1229    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1230    _configtest.c: In function 'main':\n>> 1231    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1232        8 |     (void) HAVE_DECL_SIGNBIT;\n     1233          |            ^~~~~~~~~~~~~~~~~\n     1234          |            HAVE_DECL_ISNAN\n     1235    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1817    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1818    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1819    collect2: error: ld returned 1 exit status\n     1820    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1821    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1822    collect2: error: ld returned 1 exit status",
    ">> 1819    collect2: error: ld returned 1 exit status\n     1820    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1821    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1822    collect2: error: ld returned 1 exit status\n     1823    failure.\n     1824    removing: _configtest.c _configtest.o\n     1825    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1826",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2972     2707 |         if (ret->obmeta.base == -1) {\n     2973          |                              ^~\n     2974    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2975    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2976     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2977          |                       ^\n  >> 2978    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2974    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2975    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2976     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2977          |                       ^\n>> 2978    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2980          |                        ^\n  >> 2981    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2982     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2976     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2977          |                       ^\n  >> 2978    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2980          |                        ^\n>> 2981    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2982     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2983          |                         ^\n  >> 2984    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2985     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2980          |                        ^\n  >> 2981    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2982     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2983          |                         ^\n>> 2984    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2985     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2986          |                             ^\n  >> 2987    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2983          |                         ^\n  >> 2984    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2985     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2986          |                             ^\n>> 2987    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2988     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2989          |                         ^\n  >> 2990    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2991     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2986          |                             ^\n  >> 2987    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2988     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2989          |                         ^\n>> 2990    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2991     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2992          |                          ^\n  >> 2993    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2994     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2988     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2989          |                         ^\n  >> 2990    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2991     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2992          |                          ^\n>> 2993    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2994     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2995          |                              ^\n     2996    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3139     2707 |         if (ret->obmeta.base == -1) {\n     3140          |                              ^~\n     3141    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3142    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3143     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3144          |                       ^\n  >> 3145    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3141    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3142    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3143     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3144          |                       ^\n>> 3145    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3147          |                        ^\n  >> 3148    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3149     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3143     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3144          |                       ^\n  >> 3145    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3147          |                        ^\n>> 3148    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3149     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3150          |                         ^\n  >> 3151    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3152     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3147          |                        ^\n  >> 3148    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3149     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3150          |                         ^\n>> 3151    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3152     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3153          |                             ^\n  >> 3154    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3150          |                         ^\n  >> 3151    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3152     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3153          |                             ^\n>> 3154    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3155     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3156          |                         ^\n  >> 3157    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3158     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3153          |                             ^\n  >> 3154    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3155     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3156          |                         ^\n>> 3157    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3158     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3159          |                          ^\n  >> 3160    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3161     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3155     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3156          |                         ^\n  >> 3157    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3158     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3159          |                          ^\n>> 3160    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3161     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3162          |                              ^\n  >> 3163    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -",
    "st.o -o _configtest\n     103     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 104     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 108     collect2: error: ld returned 1 exit status",
    ">> 105     collect2: error: ld returned 1 exit status\n     106     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 107     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 108     collect2: error: ld returned 1 exit status\n     109     failure.\n     110     removing: _configtest.c _configtest.o\n     111     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     112",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     161     gcc: _configtest.c\n>> 162     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     163         1 | #include <sys/endian.h>\n     164           |          ^~~~~~~~~~~~~~\n     165     compilation terminated.\n  >> 166     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "161     gcc: _configtest.c\n  >> 162     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     163         1 | #include <sys/endian.h>\n     164           |          ^~~~~~~~~~~~~~\n     165     compilation terminated.\n>> 166     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     167         1 | #include <sys/endian.h>\n     168           |          ^~~~~~~~~~~~~~\n     169     compilation terminated.\n     170     failure.\n     171     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     277     gcc: _configtest.c\n     278     _configtest.c: In function 'main':\n>> 279     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     280         7 |     (void) SIZEOF_LONGDOUBLE;\n     281           |            ^~~~~~~~~~~~~~~~~\n     282           |            SIZEOF_LONG_DOUBLE\n     283     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "280         7 |     (void) SIZEOF_LONGDOUBLE;\n     281           |            ^~~~~~~~~~~~~~~~~\n     282           |            SIZEOF_LONG_DOUBLE\n     283     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     284     _configtest.c: In function 'main':\n>> 285     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     286         7 |     (void) SIZEOF_LONGDOUBLE;\n     287           |            ^~~~~~~~~~~~~~~~~\n     288           |            SIZEOF_LONG_DOUBLE\n     289     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     423     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 424     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 425     collect2: error: ld returned 1 exit status\n     426     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 427     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 428     collect2: error: ld returned 1 exit status",
    ">> 425     collect2: error: ld returned 1 exit status\n     426     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 427     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 428     collect2: error: ld returned 1 exit status\n     429     failure.\n     430     removing: _configtest.c _configtest.o\n     431     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     432",
    "857     gcc: _configtest.c\n     858     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     859         3 | #pragma clang diagnostic error \"-Wattributes\"\n     860           |\n     861     _configtest.c: In function '__declspec':\n>> 862     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     863         5 | int __declspec(thread) foo;\n     864           |                        ^~~\n  >> 865     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     866         9 | {",
    "860           |\n     861     _configtest.c: In function '__declspec':\n  >> 862     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     863         5 | int __declspec(thread) foo;\n     864           |                        ^~~\n>> 865     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     866         9 | {\n     867           | ^\n     868     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     869         5 | int __declspec(thread) foo;",
    "866         9 | {\n     867           | ^\n     868     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     869         5 | int __declspec(thread) foo;\n     870           |     ^~~~~~~~~~\n>> 871     _configtest.c:11: error: expected '{' at end of input\n     872        11 | }\n     873           |\n     874     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     875        11 | }\n     876           |",
    "876           |\n     877     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     878         3 | #pragma clang diagnostic error \"-Wattributes\"\n     879           |\n     880     _configtest.c: In function '__declspec':\n>> 881     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     882         5 | int __declspec(thread) foo;\n     883           |                        ^~~\n  >> 884     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     885         9 | {",
    "879           |\n     880     _configtest.c: In function '__declspec':\n  >> 881     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     882         5 | int __declspec(thread) foo;\n     883           |                        ^~~\n>> 884     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     885         9 | {\n     886           | ^\n     887     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     888         5 | int __declspec(thread) foo;",
    "885         9 | {\n     886           | ^\n     887     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     888         5 | int __declspec(thread) foo;\n     889           |     ^~~~~~~~~~\n>> 890     _configtest.c:11: error: expected '{' at end of input\n     891        11 | }\n     892           |\n     893     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     894        11 | }\n     895           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-petsc4py/spack_env_dir/linux-r\n             hel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g3\n             5si6/include/python3.9 -c'\n     1223    gcc: _configtest.c\n     1224    _configtest.c: In function 'main':\n>> 1225    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1226        8 |     (void) HAVE_DECL_SIGNBIT;\n     1227          |            ^~~~~~~~~~~~~~~~~\n     1228          |            HAVE_DECL_ISNAN\n     1229    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1226        8 |     (void) HAVE_DECL_SIGNBIT;\n     1227          |            ^~~~~~~~~~~~~~~~~\n     1228          |            HAVE_DECL_ISNAN\n     1229    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1230    _configtest.c: In function 'main':\n>> 1231    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1232        8 |     (void) HAVE_DECL_SIGNBIT;\n     1233          |            ^~~~~~~~~~~~~~~~~\n     1234          |            HAVE_DECL_ISNAN\n     1235    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1817    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1818    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1819    collect2: error: ld returned 1 exit status\n     1820    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1821    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1822    collect2: error: ld returned 1 exit status",
    ">> 1819    collect2: error: ld returned 1 exit status\n     1820    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1821    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.1-cxiueueitegl\n             5brymwlc3v6sqfvhyld2/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1822    collect2: error: ld returned 1 exit status\n     1823    failure.\n     1824    removing: _configtest.c _configtest.o\n     1825    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1826",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2972     2707 |         if (ret->obmeta.base == -1) {\n     2973          |                              ^~\n     2974    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2975    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2976     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2977          |                       ^\n  >> 2978    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2974    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2975    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2976     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2977          |                       ^\n>> 2978    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2980          |                        ^\n  >> 2981    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2982     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2976     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2977          |                       ^\n  >> 2978    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2980          |                        ^\n>> 2981    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2982     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2983          |                         ^\n  >> 2984    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2985     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2979     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2980          |                        ^\n  >> 2981    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2982     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2983          |                         ^\n>> 2984    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2985     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2986          |                             ^\n  >> 2987    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2983          |                         ^\n  >> 2984    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2985     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2986          |                             ^\n>> 2987    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2988     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2989          |                         ^\n  >> 2990    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2991     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2986          |                             ^\n  >> 2987    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2988     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2989          |                         ^\n>> 2990    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2991     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2992          |                          ^\n  >> 2993    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2994     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2988     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2989          |                         ^\n  >> 2990    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2991     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2992          |                          ^\n>> 2993    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2994     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2995          |                              ^\n     2996    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3139     2707 |         if (ret->obmeta.base == -1) {\n     3140          |                              ^~\n     3141    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3142    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3143     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3144          |                       ^\n  >> 3145    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3141    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3142    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3143     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3144          |                       ^\n>> 3145    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3147          |                        ^\n  >> 3148    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3149     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3143     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3144          |                       ^\n  >> 3145    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3147          |                        ^\n>> 3148    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3149     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3150          |                         ^\n  >> 3151    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3152     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3146     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3147          |                        ^\n  >> 3148    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3149     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3150          |                         ^\n>> 3151    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3152     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3153          |                             ^\n  >> 3154    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3150          |                         ^\n  >> 3151    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3152     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3153          |                             ^\n>> 3154    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3155     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3156          |                         ^\n  >> 3157    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3158     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3153          |                             ^\n  >> 3154    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3155     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3156          |                         ^\n>> 3157    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3158     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3159          |                          ^\n  >> 3160    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3161     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3155     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3156          |                         ^\n  >> 3157    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3158     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3159          |                          ^\n>> 3160    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3161     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3162          |                              ^\n  >> 3163    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -"
]